js/app.js

```diff
@@ -0,0 +1,163 @@
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a < b < c);
+console.log(a <= a <= d);
+console.log(e === 'Kevin');
+console.log(48 !== '48');
+
+let letters = "A";
+let i = 0;
+
+while (i < 20) {
+	letters += "A";
+	i++;
+}
+
+console.log(letters);
+
+
+
+
+
+
+
+//Section 1
+//Question 1: interpolation is substituting a variable for a value (ex. Changing "Please" + " believe." to Please" + word where word is a variable that was substituted in to hold the string value " believe.")
+	//concatenation is combining two strings together (ex. "Please" + " believe." evaluates to "Please believe.")
+
+//Question 2:  DRY stands for Don't Repeat Yourself. It's important for writing code efficiently and in a way that is easy to read.
+	//We can use loops and functions to help us reuse code and keep our code DRY.
+
+
+//Section 2
+// a < b
+// c > d
+// 'Name' === 'Name'
+// a < b < c
+// a <= a <= d
+// e === 'Kevin'
+// 48 !== '48'
+
+//Section 3
+//Loop 1
+	//The loop will be infinite. true will always be true.
+//Loop 2
+	//It's not infinite. It will output 'Do not run this loop' once.
+//Loop 3
+	//Expected result: AAAAAAAAAAAAAAAAAAAAA
+	//Actual result:   AAAAAAAAAAAAAAAAAAAAA 
+
+
+//Section 4
+//Question 1
+	//for loops are more compact because they have the initialization and incrementation of the loop variable all in the loop statement.
+	//while loops can handle randomness better if you're doing different thing to the loop variable with each loop.
+	//They both have a loop variable that gets initialized and a condition that gets checked every time before looping.
+	//When they aren't infinite, they'll also do something to the loop variable that will eventual make the loop condition false.
+
+//Question 2
+
+	for (let i=0;i<=999;i++) {
+		console.log(i);
+	}
+
+//Question 3
+	//The first part of the control statement is: the initialization
+	//The second part of the control statement is: the condition
+	//The third part of the control statement is: the final expression
+
+//Question 4
+
+	for (let i=999;i>=0;i--) {
```

Comment:
claustrophobia! why are you leavingoutallthespacesleavesomespacessoitsmorelegible